bin/oio-blob-rebuilder

```diff
@@ -50,8 +50,12 @@ def make_arg_parser():
     ifile_help = "Read chunks from this file instead of rdir. " \
                  "Each line should be formatted like " \
                  "'container_id|content_id|short_chunk_id'."
-    parser.add_argument('input_file', nargs='?',
+    parser.add_argument('--input_file', nargs='?',
                         help=ifile_help)
+    parser.add_argument('--beanstalkd', nargs='?',
+                        help='address of the service beanstalkd')
+    parser.add_argument('--beanstalkd_tube', nargs='?',
+                        help='the tube to use for the service beanstalkd')
```

Comment:
The `nargs` are useless. Besides, I advise you not to use underscores in argument keys.